src/sql/src/kafka_util.rs

```diff
@@ -11,14 +11,18 @@
 
 use std::collections::BTreeMap;
 use std::convert;
+use std::convert::TryInto;
```

Comment:
tiny nit: I think we would usually try to write these imports like `use std::convert::{self, TryInto};`

Reply:
Thanks and done! I _thought_ these were merged automatically by `rustfmt`, but I apparently was confused. 🙈

Reply:
`rustfmt` will reorder your imports! but it doesn't fix these things, unfortunately. (I've learned the hard way! 😂 )